src/os/bluestore/BlueStore.h

```diff
@@ -2061,6 +2063,10 @@ class BlueStore : public ObjectStore,
   void _set_alloc_sizes();
   void _set_blob_size();
   void _set_finisher_num();
+  void _update_osd_memory_options();
+  void _update_osd_memory_target_size();
+  void _set_config_changed();
```

Comment:
nit: IMO this methods are an overkill now...

Reply:
I have removed them.